### PR TITLE
Fix testing

### PR DESCRIPTION
**Pull request by tobias-urdin**

### CODE REVIEW

**Comment by tobias-urdin** [comment]
Need to investigate manually why test is hanging.

File: .mergify.yml [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+pull_request_rules:
+  - name: automatic merge for master
+    actions:
+      merge:
+        method: rebase
+        rebase_fallback: merge
+        strict: true
+    conditions:
+    - label!=work-in-progress
+    - '#approved-reviews-by>=2'
```
Comment (jd) [inline .mergify.yml:10]: maybe 1 is enough nowadays :)

**Review by jd — APPROVED** [review]
LGTM

**Comment by tobias-urdin** [comment]
@jd this is OK, the failing test is from old travis-ci.org, the new ones passed

File: .mergify.yml [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+  - name: automatic merge for master
+    actions:
+      merge:
+        method: rebase
+        rebase_fallback: merge
+        strict: true
+    conditions:
+    - label!=work-in-progress
+    - '#approved-reviews-by>=1'
+    - status-success=continuous-integration/travis-ci/pr
```
Comment (jd) [inline .mergify.yml:11]:
```suggestion
    - status-success=Travis CI - Pull Request
```

File: .mergify.yml [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
+  - name: automatic merge for stable branches
+    actions:
+      merge:
+        method: rebase
+        rebase_fallback: merge
+        strict: true    
+    conditions:
+    - label!=work-in-progress
+    - '#approved-reviews-by>=1'
+    - status-success=continuous-integration/travis-ci/pr
```
Comment (jd) [inline .mergify.yml:22]:
```suggestion
    - status-success=Travis CI - Pull Request
```

**Review by jd — CHANGES_REQUESTED** [review]
The CI is wrong for Mergify.

**Comment by tobias-urdin** [comment]
Don't think so but we can try, the issue does not exist in gnocchi repo

**Comment by jd** [comment]
> Don't think so but we can try, the issue does not exist in gnocchi repo

it does, nothing will get automerged until you fix it

**Pull request merged**